cmplxs/cmplxs_test.go

```diff
@@ -725,6 +725,62 @@ func TestMul(t *testing.T) {
 	}
 }
 
+func TestMulConj(t *testing.T) {
```

Comment:
I think this could also include a test that `s2` is not modified.

Reply:
Done